fe/fe-core/src/main/java/com/starrocks/qe/SessionVariable.java

```diff
@@ -2250,6 +2249,9 @@ public SessionVariable setHiveTempStagingDir(String hiveTempStagingDir) {
     @VarAttr(name = CONNECTOR_INCREMENTAL_SCAN_RANGE_SIZE)
     private int connectorIncrementalScanRangeSize = 500;
 
+    @VarAttr(name = ENABLE_CONNECTOR_ASYNC_LIST_PARTITIONS)
+    private boolean enableConnectorAsyncListPartitions = false;
```

Comment:
why false default? when set to true?

Reply:
@Youngwb I think we can enable this feature later.  And i've seen the above comments and he gets his point:  async partition does not work consistently with MV, which probably confuses user.